collaboration/constraints.md

```diff
@@ -1,34 +1,36 @@
-<!-- this template is for inspiration, feel free to change it however you like! -->
 
 # Constraints
 
 Some boundaries around our project.
 
-## External
-
-<!--
-  constraints coming from the outside that your team has no control over:
-  - project deadlines
-  - number of unit tests required to pass a code review
-  - technologies (sometimes a client will tell you what to use)
-  - power or connectivity
-  - ...
--->
-
-## Internal: Involuntary
-
-<!--
-  constraints that come from within your team, and you have no control over:
-  - each of your individual skill levels
-  - amount of time available to work on the project
--->
-
-## Internal: Voluntary
-
-<!--
-  constraints that your team decided on to help scope the project. they may include:
-  - coding style & conventions
-  - agree on a code review checklist for the project repository
-  - the number of hours you want to spend working
-  - only using the colors black and white
--->
+## 1. External Constraints
+
+- **Program Requirements**
+  The CDS program's expectations for group collaboration and
+  deliverables, such
+  as using GitHub for documentation.
+- **Time Zones**
+  Group members are located in different parts of the world, making it
+  challenging to find universally convenient meeting times.
+- **Communication Tools**
+  Limitations of WhatsApp and Slack for certain tasks like file sharing or code reviews.
```

Comment:
Could you be more specific about how the communication tools pose as external constraints?

Reply:
But maybe it'd be more clear if you put an example like: `difficulty in sharing video files of a member reviewing the code or explaining some topics`.